Gopkg.toml

```diff
@@ -143,3 +143,7 @@ required = [
 [[constraint]]
   name = "github.com/docker/distribution"
   version = "v2.7.1"
+
+[[constraint]]
```

Comment:
I am confused, this doesn't seem to be used anywhere. But the presubmit that checks the dependencies isn't failing either..?

Reply:
I honestly didn't change that manually, the dep cli did, so not quite sure either